server/AccountManagement.java

```diff
@@ -29,14 +29,15 @@ public String newAccount(CustomerID customer, NewBankClientHandler newBankClient
             }
             //if account name already exists, error message and prompts new input,
             //else exit while loop
-            if (!NewBank.getBank().getCustomers().get(customer.getKey()).checkExistingAccount(newAccountName)){
+            if (!customer.checkExistingAccount(newAccountName)) {
                 valid = true;
-            }else{
+            }
+            else{
                 newBankClientHandler.sendOutput("Account name is taken, please try again.");
             }
         }
         //creates new account, defaults to balance of 0.0
-        NewBank.getBank().getCustomers().get(customer.getKey()).addAccount(new Account(newAccountName, 0.0));
+        customer.addAccount(new newbank.server.Account(newAccountName, 0.0));
```

Comment:
`customer.addAccount(new newbank.server.Account(newAccountName, 0.0));` can be changed to just `customer.addAccount(new Account(newAccountName, 0.0));`

The `newbank.server` is not necessary, and actually makes the code harder to read, please remove it everywhere in this pull request